apps/src/aichat/views/teacherFeedback/teacher-feedback-footer.module.scss

```diff
@@ -0,0 +1,40 @@
+@import 'color.scss';
+@import 'font.scss';
+@import '@cdo/apps/componentLibrary/typography/typography.module';
+
+.teacherFeedbackContainer {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  justify-content: flex-end;
+
+  .button i {
+    color: $light-gray-600;
```

Comment:
Why do we have to specify these explicitly vs a DSCO default? Matching AI TA?

Reply:
Yeah, just matched the constants in AI Diff chat - I'll run this by Mark to confirm